### ISSUE TITLE
Unpin ipywidgets and ipympl, or pin them to newer versions.

### ISSUE THREAD

**Issue by danielballan** [nsls-ii-forge/analysis-feedstock#22]
The current pins are very old, and ipympl in particular has improved a lot.

**Comment by mrakitin** [comment]
It's to keep them consistent with the NSLS-II Jupyterhub.

**Comment by danielballan** [comment]
Good point. We have dug ourselves into a bad hole here. Our deployment is based on the flawed assumption that you can use the same front-end code (as specified by the jupyter server environment on the `jupyterhub` VM) with any back-end code (as specified by a given kernel on beamline machine).

As long as beamlines sometimes rely on very old kernels, we are stuck with very old front-end code to remain compatible, in particular with interactive matplotlib figures.

The new model that selects between _images_ rather than _kernels_ keeps front-end and back-end in sync. For beamlines that run their our jupyter servers (e.g. LIX) we may need to create a special case so that they can leverage the latest features in ipympl.
